keyboards/ergodox_ez/keymaps/dvorak_42_key/keymap.c

```diff
@@ -61,7 +70,7 @@ enum custom_keycodes {
 #define KEYSEL           3 // arrow navigation + shift (allow text selection)
```

Comment:
You may actually want to convert this block to an enum, as it may be easier to maintain.

Reply:
thank you, i'll do it with my next change